app/src/androidTest/java/com/github/campus_capture/bootcamp/SectionColorsTest.java

```diff
@@ -1,23 +1,45 @@
 package com.github.campus_capture.bootcamp;
 
-import static com.github.campus_capture.bootcamp.authentication.Section.*;
+import static com.github.campus_capture.bootcamp.authentication.Section.AR;
+import static com.github.campus_capture.bootcamp.authentication.Section.CGC;
+import static com.github.campus_capture.bootcamp.authentication.Section.EL;
+import static com.github.campus_capture.bootcamp.authentication.Section.GC;
+import static com.github.campus_capture.bootcamp.authentication.Section.GM;
+import static com.github.campus_capture.bootcamp.authentication.Section.IN;
+import static com.github.campus_capture.bootcamp.authentication.Section.MA;
+import static com.github.campus_capture.bootcamp.authentication.Section.MT;
+import static com.github.campus_capture.bootcamp.authentication.Section.MX;
+import static com.github.campus_capture.bootcamp.authentication.Section.NONE;
+import static com.github.campus_capture.bootcamp.authentication.Section.PH;
+import static com.github.campus_capture.bootcamp.authentication.Section.SC;
+import static com.github.campus_capture.bootcamp.authentication.Section.SIE;
+import static com.github.campus_capture.bootcamp.authentication.Section.SV;
```

Comment:
That's odd lol is there a reason for this ?

Reply:
Oh you are right, I did not see that. I change this right now!